.travis.yml

```diff
@@ -4,10 +4,6 @@ cache: pip
 
 matrix:
     include:
-        - python: 2.7
-          env: TOXENV=py27
-        - python: 3.4
-          env: TOXENV=py34
         - python: 3.5
```

Comment:
can you add python 3.8 here?

Reply:
that broke the build, looks like pygame system deps need to be installed. Either add those or remove py3.8 again and we look at it later?

Reply:
removed